app/models/user.rb

```diff
@@ -3,6 +3,7 @@ class User < ApplicationRecord
 
   validates_uniqueness_of :email
 
+	has_many :orders
```

Comment:
this is another part. I think I am going to just wait till will has his part as well so that we are not re doing each others work .